android/src/main/java/com/stremio/vlc/video/VLCVideoProps.java

```diff
@@ -8,8 +8,8 @@
     static final int MEDIA_START_TIME_DEFAULT_VALUE = 0;
     static final String MEDIA_AUTOPLAY_PROP = "autoplay";
     static final boolean MEDIA_AUTOPLAY_DEFAULT_VALUE = true;
-    static final String MEDIA_HW_DECODER_ENABLED_PROP = "hwDecoderEnabled";
-    static final boolean MEDIA_HW_DECODER_ENABLED_DEFAULT_VALUE = true;
+    static final String MEDIA_HW_DECODER_ENABLED_PROP = "hwDecoderMode";
```

Comment:
rename those constants to match the new name

Reply:
done